tests/test_runner/test_hooks.py

```diff
@@ -317,6 +317,55 @@ def test_one_cycle_runner_hook():
     hook.writer.add_scalars.assert_has_calls(calls, any_order=True)
 
 
+def test_onecycle_lr_updater_hook():
```

Comment:
Maybe  `def test_onecycle_lr_updater_hook` can be merged into `def test_one_cycle_runner_hook()`